src/pure.js

```diff
@@ -44,7 +49,7 @@ export function createConsole({ isSilent: defaultIsSilent = true } = {}) {
         .toString('utf8')
         // Strip out the new line character in the end
         .slice(0, -1);
-      logs.push([currentLevel, message]);
+      logs.push([currentLevel, stripsAnsi ? stripAnsi(message) : message]);
       if (currentLevel && currentLevel in levels) {
         levels[currentLevel] = [levels[currentLevel], message]
```

Comment:
Should also strip ansi here in `levels` and also `records` below. Would help a lot if can also add tests for them.

Reply:
@kevin940726 Fixed

>Would help a lot if can also add tests for them.

I've added some minimum testing for levels and records. Does it fit you?

FWIW, I'm not sure how the `records` is really supposed to be used (as it's not really documented, nor explicitly tested).

Reply:
Huh, I forgot what it is either. It doesn't hurt to also handle them though 😉 .